bin/build_fixups.sh

```diff
@@ -45,6 +51,9 @@ $SUDO rm -f $AOMP/examples/raja/*.sh
 
 # Clean libexec
 $SUDO rm -rf $AOMP/libexec
+if [[ ! "$DEVEL_PACKAGE" =~ "devel" ]]; then
+  $SUDO rm -rf $AOMP/share
+fi
```

Comment:
This is not needed due to being handled in the build_openmp_extras.sh script.

Reply:
Removed the code snippet